pairing.md

```diff
@@ -0,0 +1,57 @@
+# Pairing
+
+Pair programming is the process of two people sharing a development environment to solve a technical problem. Outside of programming, pairing can be a used to solve all manner of diverse problems. 
+
+## When should we pair?
+
+There's no concrete right or wrong time, but it's worth keeping in mind that the way you pair needs modifying to make sure you're achieving that goal. At HX, we would typically use pairing to onboard new hires, solve difficult problems and spread knowledge. 
+
+The Scrum Alliance describe the pair as a Driver, in control of the development environment and a Navigator providing instruction. This slightly over simplifies the relationship, so we'll break out some of the different ways we pair depending on the situation.
+
+### Onboarding
+
+During onboarding you would typically pair a new hire with an experienced engineer. The intention is for the experienced engineer to provide the context and safety net needed for the new hire to learn on a real platform. It has been proven, externally and internally, to be a really effective, supportive way of getting a new hire up to speed.
```

Comment:
Could you replace 'new hire' with 'new starter' throughout? (not a big deal, but 'starter' sounds more like a HX term than 'hire' does)